internal/commands/user/enable_test.go

```diff
@@ -0,0 +1,327 @@
+package user
+
+import (
+	"errors"
+	"strings"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestEnableSetup(t *testing.T) {
+	t.Run("should construct a realm client with the configured base url", func(t *testing.T) {
+		profile := mock.NewProfile(t)
+		profile.SetRealmBaseURL("http://localhost:8080")
+		cmd := &CommandEnable{inputs: enableInputs{}}
+
+		assert.Nil(t, cmd.realmClient)
+		assert.Nil(t, cmd.Setup(profile, nil))
+		assert.NotNil(t, cmd.realmClient)
+	})
+}
+
+func TestEnableHandler(t *testing.T) {
+	projectID := "projectID"
+	appID := "appID"
+	testApp := realm.App{
+		ID:          appID,
+		GroupID:     projectID,
+		ClientAppID: "eggcorn-abcde",
+		Name:        "eggcorn",
+	}
+	testUsers := []realm.User{
+		{
+			ID:         "user-1",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeAnonymous}},
+			Disabled:   true,
+		},
+		{
+			ID:         "user-2",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeAnonymous}},
+			Disabled:   false,
+		},
+	}
+	t.Run("should enable a user when a user id is provided", func(t *testing.T) {
+		var capturedAppFilter realm.AppFilter
+		var capturedFindProjectID, capturedFindAppID string
+		var capturedEnableProjectID, capturedEnableAppID string
+		realmClient := mock.RealmClient{}
+		realmClient.FindAppsFn = func(filter realm.AppFilter) ([]realm.App, error) {
+			capturedAppFilter = filter
+			return []realm.App{testApp}, nil
+		}
+		realmClient.FindUsersFn = func(groupID, appID string, filter realm.UserFilter) ([]realm.User, error) {
+			capturedFindProjectID = groupID
+			capturedFindAppID = appID
+			return testUsers[:1], nil
+		}
+		realmClient.EnableUserFn = func(groupID, appID, userID string) error {
+			capturedEnableProjectID = groupID
+			capturedEnableAppID = appID
+			return nil
+		}
+		cmd := &CommandEnable{
+			inputs: enableInputs{
+				ProjectInputs: app.ProjectInputs{
+					Project: projectID,
+					App:     appID,
+				},
+				Users: []string{testUsers[0].ID},
+			},
+			realmClient: realmClient,
+		}
+
+		assert.Nil(t, cmd.Handler(nil, nil))
+		assert.Equal(t, realm.AppFilter{App: appID, GroupID: projectID}, capturedAppFilter)
+		assert.Equal(t, projectID, capturedFindProjectID)
+		assert.Equal(t, appID, capturedFindAppID)
+		assert.Equal(t, projectID, capturedEnableProjectID)
+		assert.Equal(t, appID, capturedEnableAppID)
+		assert.Equal(t, testUsers[0].ID, cmd.inputs.Users[0])
+		assert.Nil(t, cmd.outputs[0].err)
+		assert.Equal(t, cmd.outputs[0].user, testUsers[0])
+	})
+
+	t.Run("should save failed enable errors", func(t *testing.T) {
```

Comment:
i think a lot of the comments i made in `disable_test` are going to end up applying here (so i'll avoid repeating a few of the nits), but this test looks like it can be tabled with the one above